antora-ui/supplemental-ui/css/isyfact.css

```
@@ -176,10 +196,17 @@ main .content .doc {
 }
 
 @media screen and (min-width: 1920px) {
-    aside.toc.sidebar {
-        flex-basis: 20rem;
+    .nav-container{
+        width: 20rem;
     }
-    .doc {
-        margin: 0 2rem;
+    main .content .doc{
+        max-width: 1400px;
+        /*left: auto;*/
+        /*right: auto;*/
+    }
+
+    aside.toc.sidebar {
+        padding: 1rem;
+        max-width: 20rem;
```

max-width: 20rem bewirkt keine Veränderung, da in Z.173 flex-basis: 15rem festgesetzt wurde. Evtl. flex-basis: 20rem;?

```suggestion
    aside.toc.sidebar {
        flex-basis: 20rem;
```